src/server/handler/subscriberHandler.ts

```diff
@@ -18,19 +18,22 @@ export default class SubscriberHandler extends Handler {
 
     private async getAll(req: express.Request, res: express.Response): Promise<void> {
         try {
-            const subscriber = await Subscriber.find({userId: req.query.token}).populate("app");
+            // @ts-ignore
```

Comment:
why is this here? consider using https://github.com/hopperteam/hopper-server/blob/master/src/handler/authMiddleware.ts#L10-L16